### ISSUE TITLE
Document frame rate with your 128x96 panel.

### ISSUE THREAD

**Issue by philipag** [hpwit/I2SClocklessVirtualLedDriver#23]
Great detail in your readme but would be nice to know what frame rates you achieve with your panels.

**Comment by hpwit** [comment]
Hello
For my 128x96 I have 48 16x16 panels. Each panel has it's own virtual pin. 
So the fps is the fps of 256 ws2812.=> 127 fps. The max theorical gps for 256 less is 130fps. I achieve 127 fps due to overhead in my driver. 
Hence the 12288 are refreshed at 127fps.

**Comment by philipag** [comment]
Thanks!

**Comment by hpwit** [comment]
Are you looking at using this driver for one of your builds?

**Comment by philipag** [comment]
Yes for a 3mx3m. 90x90 pixel panel art project.  But it wil have to wait since this will be a hobby project.
